winit/src/conversion.rs

```diff
@@ -1112,7 +1112,7 @@ pub fn native_key_code(
     }
 }
 
-/// Converts some [`UserAttention`] into it's `winit` counterpart.
+/// Converts some [`UserAttention`] into its `winit` counterpart.
```

Comment:
This is technically out of scope, but I noticed this typo so I figured I'd fix it while I'm at it.